cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt

```diff
@@ -0,0 +1,513 @@
+
+    private var mSaveMode: SaveMode                        = SaveMode(STANT_MODE)
+
+    private val currentImagesQuantity: Int
+        get() = mCameraPhotosAdapter?.itemCount ?: 0
+
+    private val isOverLimit: Boolean
+        get() = if (mPhotosLimit == -1) {
+            false
+        } else mImageListSize + currentImagesQuantity > mPhotosLimit
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        activity?.window?.setFlags(
+                WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED,
+                WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED)
+
+        context?.let {
+            setupDialog(it)
+
+            mPath                = ImageViewFileUtil.getPrivateTempDirectory(it)
+            mDialogLoader        = DialogLoader(it)
+            mImageGenerator      = ImageGenerator(it)
+            mCameraPhotosAdapter = CameraPhotosAdapter(it, this)
+        }
+    }
+
+    private fun setupDialog(context: Context) {
+        mPreviewPicDialog = Dialog(context, android.R.style.Theme_Black_NoTitleBar_Fullscreen)
+
+        mCameraPhotoPreviewDialogBinding = DataBindingUtil.inflate(LayoutInflater.from(context),
+                R.layout.camera_photo_preview_dialog, null, false)
+
+        mCameraPhotoPreviewDialogBinding.handler = this
+        mPreviewPicDialog?.setContentView(mCameraPhotoPreviewDialogBinding.root)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        mCameraFragmentBinding = DataBindingUtil.inflate(inflater, R.layout.camera_fragment, container, false)
+
+        if (hasNavigationBar()) {
+            setNavigationCameraControlsPadding()
+        }
+
+        return mCameraFragmentBinding.root
+    }
+
+    private fun hasNavigationBar(): Boolean {
+        val id = resources.getIdentifier("config_showNavigationBar", "bool", "android")
+        return id > 0 && resources.getBoolean(id)
+    }
+
+    private fun setNavigationCameraControlsPadding() {
+        ofNullable(mCameraFragmentBinding.cameraFragmentBottomLinearLayout).executeIfPresent { navigationCamera ->
+            navigationCamera.setPadding(
+                    navigationCamera.paddingLeft,
+                    navigationCamera.paddingTop,
+                    navigationCamera.paddingRight,
+                    navigationCamera.bottom + convertDpToPixels(48))
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        mSaveOnlySnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_save_only_mode_message),
+                Snackbar.LENGTH_SHORT)
+        mStantSaveModeSnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_stant_save_mode_message),
```

Comment:
change this strings to be dinamically, texts should be provided by those who use the library

Reply:
Fixed